README.md

```diff
@@ -1,5 +1,4 @@
 # FCC Test Suite Template
-
 **Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
 You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it..
```

Comment:
I think it would be better to make changes and add valuable information about your project in this file. you can follow this example https://github.com/microverseinc/readme-template

Reply:
Good job suggesting a link as a resource to better the code. 🚀 However, a good review practice is to first commend and use polite words. 

For example instead of  `"I think it would be better"` you could comment `"Kindly consider..."` or `"I would want to suggest that..."` 👍🏿 

- [ ] Please ensure to make your words more polite. 💯

Reply:
Thanks for your suggestion bro but my words are also polite but I will use your words as per your required changes